pkg/scheduler/scheduler.go

```diff
@@ -646,8 +649,41 @@ func (s *HivedScheduler) preemptRoutine(args ei.ExtenderPreemptionArgs) *ei.Exte
 		panic(internal.NewBadRequestError(fmt.Sprintf(
 			"Pod has already been binding to node %v",
 			podStatus.Pod.Spec.NodeName)))
-	} else if podStatus.PodState == internal.PodPreempting {
-		victims := podStatus.PodScheduleResult.PodPreemptInfo.VictimPods
```

Comment:
Do we still need to record the preemption result in filterRountine now?

Reply:
No need in fact, but just keep it there for consistent and future usage.

---
In reply to: [398993753](https://github.com/microsoft/hivedscheduler/pull/11#discussion_r398993753) [](ancestors = 398993753)